### ISSUE TITLE
render dashboard report via Docker

### ISSUE THREAD

**Issue by mvarewyck** [inbo/reporting-rshiny-grofwildjacht#353]
When trying to knit the report I get an error. See log:

[docker_log_dashboard_knit_error.txt](https://github.com/inbo/reporting-rshiny-grofwildjacht/files/9926495/docker_log_dashboard_knit_error.txt)

![image](https://user-images.githubusercontent.com/16779320/199672119-ea2fb08c-31e8-4f19-9123-35f00be438a2.png)

_Originally posted by @SanderDevisscher in https://github.com/inbo/reporting-rshiny-grofwildjacht/pull/346#pullrequestreview-1166432060_

**Comment by mvarewyck** [comment]
Some dependencies were missing, already added to dockerfile.
Currently, the report renders, but the plotly graphs selected by the user are not.

**Comment by mvarewyck** [comment]
Should get fixed by using `rmarkdown` version >=2.14 instead of 2.8

**Comment by mvarewyck** [comment]
In the end it was a combination of plotly, magick and rmarkdown

```
RUN R -q -e "remotes::install_version('plotly', version = '4.10.1', repos = 'http://cran.us.r-project.org', upgrade = 'never')" 
RUN R -q -e "remotes::install_version('rmarkdown', version = '2.18', repos = 'http://cran.us.r-project.org', upgrade = 'never')"
RUN R -q -e "remotes::install_version('magick', version = '2.7.3', repos = 'http://cran.us.r-project.org', upgrade = 'never')"
```
